mmpose/datasets/datasets/body3d/body3d_mpi_inf_3dhp_dataset.py

```diff
@@ -360,7 +360,7 @@ def _report_3d_auc(self, keypoint_results, mode='3dauc'):
 
         preds = np.stack(preds)
         gts = np.stack(gts)
-        masks = np.ones_like(gts[:, :, 0], dtype=np.bool)
+        masks = np.ones_like(gts[:, :, 0], dtype=bool)
```

Comment:
why these changes?

Reply:
The original code raises DeprecationWarning with a reminder that changing like this will not change the behavior.